apps/nsq_to_http/nsq_to_http.go

```diff
@@ -49,14 +50,17 @@ var (
 	statusEvery        = flag.Int("status-every", 250, "the # of requests between logging status (per handler), 0 disables")
 	contentType        = flag.String("content-type", "application/octet-stream", "the Content-Type used for POST requests")
 
-	getAddrs         = app.StringArray{}
-	postAddrs        = app.StringArray{}
-	nsqdTCPAddrs     = app.StringArray{}
-	lookupdHTTPAddrs = app.StringArray{}
+	getAddrs           = app.StringArray{}
+	postAddrs          = app.StringArray{}
+	customHeaders      = app.StringArray{}
+	nsqdTCPAddrs       = app.StringArray{}
+	lookupdHTTPAddrs   = app.StringArray{}
+	validCustomHeaders map[string]string
 )
 
 func init() {
 	flag.Var(&postAddrs, "post", "HTTP address to make a POST request to.  data will be in the body (may be given multiple times)")
+	flag.Var(&customHeaders, "custom-header", "Custom header for HTTP requests (may be given multiple times)")
```

Comment:
I suggest making the flag shorter, just `header` (it would also match `curl`'s `--header` flag)

Reply:
Agreed, this would enable people who are comfortable with curl to use this option

Reply:
Fixed